src/services/products_service.ts

```diff
@@ -3,22 +3,17 @@ import { get_products, get_products_by_page } from "../api/products";
 import { productsStore } from "../stores/products";
 
 export async function getProducts(company_id: string) {
-  // Get first N products and load meta data
   const response = await get_products(company_id);
 
-  // Iterate over all products pages and update products store
-  for (let page = 1; page <= (browser ? response.meta.pages : 2); page++) {
-    const response = await get_products_by_page(company_id, page);
+  productsStore.dic.update((prev) => ({
+    ...prev,
+    ...response.data,
+  }));
 
-    console.log(response.data);
+  // // Iterate over all products pages and update products store
```

Comment:
No pude entender cómo funciona este "for", por alguna razón no funciona y devuelve "Request failed with status code 500"

Reply:
Este era para traer todos los productos.
Y tartar de renderizar la pagina de images con todos los productos (con images.) Hay que pensar cómo vamos a manejar este.
Debería haber una via más eficaz.